app/src/main/res/values/strings.xml

```diff
@@ -230,8 +230,8 @@
     <string name="replace_recovery_phrase">Replace recovery phrase</string>
     <string name="scan_your_browser_extension">Scan your browser extension</string>
     <string name="scan_browser_extension_address">Scan Browser Extension Address</string>
-    <string name="replace_recovery_phrase_description">In order to replace the recovery phrase for this safe you need to scan the address of the browser extension.</string>
-    <string name="browser_extension_not_owner">Browser extension is not an owner of this safe</string>
+    <string name="replace_recovery_phrase_description">In order to replace the recovery phrase for this Safe you need to scan the address of the browser extension.</string>
+    <string name="browser_extension_not_owner">Browser extension is not a signer of this Safe</string>
```

Comment:
We should probably also add a ticket to check all strings for:
- in use
- hardcoded strings
- owners -> signer (or whatever we decide)
- always capital Safe

This could be part of the issue for including the translations stuff

Reply:
Opened #244